sklearn/linear_model/least_angle.py

```diff
@@ -1338,6 +1359,15 @@ class LassoLarsIC(LassoLars):
         alphas. The alpha which has the smallest information criteria
         is chosen.
 
+    ``best_score_`` : float
```

Comment:
I suppose that Andreas Mueller should answer this but it seems what he meant is the ticks *``* are not needed. It is to bold the text., but now it is done by default for the attributes. Hope it helps